tutorials/Tutorial6_Better_Retrieval_via_EmbeddingRetrieval.ipynb

```diff
@@ -0,0 +1,446 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "bEH-CRbeA6NU"
+   },
+   "source": [
+    "# Better Retrieval via \"Embedding Retrieval\"\n",
+    "\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/deepset-ai/haystack/blob/master/tutorials/Tutorial6_Better_Retrieval_via_EmbeddingRetrieval.ipynb)\n",
+    "\n",
+    "### Importance of Retrievers\n",
+    "\n",
+    "The Retriever has a huge impact on the performance of our overall search pipeline.\n",
+    "\n",
+    "\n",
+    "### Different types of Retrievers\n",
+    "#### Sparse\n",
+    "Family of algorithms based on counting the occurrences of words (bag-of-words) resulting in very sparse vectors with length = vocab size.\n",
+    "\n",
+    "**Examples**: BM25, TF-IDF\n",
+    "\n",
+    "**Pros**: Simple, fast, well explainable\n",
+    "\n",
+    "**Cons**: Relies on exact keyword matches between query and text\n",
+    " \n",
+    "\n",
+    "#### Dense\n",
+    "These retrievers use neural network models to create \"dense\" embedding vectors. Within this family there are two different approaches: \n",
+    "\n",
+    "a) Single encoder: Use a **single model** to embed both query and passage.  \n",
+    "b) Dual-encoder: Use **two models**, one to embed the query and one to embed the passage\n",
+    "\n",
+    "**Examples**: REALM, DPR, Sentence-Transformers\n",
+    "\n",
+    "**Pros**: Captures semantic similarity instead of \"word matches\" (e.g. synonyms, related topics ...)\n",
+    "\n",
+    "**Cons**: Computationally more heavy, initial training of model\n",
```

Comment:
I think we can explain better what does "initial training of model" means. AFAIK there is a widespread misunderstanding in our users that NLP models must always be trained before hand and they don't know about pre-trained models. So maybe it's worth mentioning it here as well, to reinforce the concept :slightly_smiling_face:

Reply:
Good point, how about something like this?

>**Cons**: Computationally more heavy to use, initial training of model (though this is less of an issue nowadays as we can grab and use one of the many pre-trained models available i.e. we don't usually need to train the model).

Reply:
I'll leave the judgement to @mkkuemmel here :grin:

Reply:
I think that's a good solution, yes! :)